mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -2297,15 +2303,13 @@ pub async fn send_raw_transaction(mm: &MarketMakerIt, coin: &str, tx: &str) -> J
     json::from_str(&request.1).unwrap()
 }
 
-pub async fn my_balance(mm: &MarketMakerIt, coin: &str) -> Json {
-    let request = mm
-        .rpc(&json!({
-            "userpass": mm.userpass,
-            "method": "my_balance",
-            "coin": coin
-        }))
-        .await
-        .unwrap();
+pub fn my_balance(mm: &MarketMakerIt, coin: &str) -> MyBalanceResponse {
+    let request = block_on(mm.rpc(&json!({
+        "userpass": mm.userpass,
+        "method": "my_balance",
+        "coin": coin
+    })))
+    .unwrap();
```

Comment:
Could you please make it `async` back? It might be used in some other `async` test helpers and also in WASM environment, which doesn't support `block_on` at all.